spec/Vehicle/Vehicle.vspec

```diff
@@ -184,16 +184,41 @@ Speed:
   description: Vehicle speed.
 
 TravelledDistance:
+  deprecation: V3.1 moved to Vehicle.TraveledDistance
   datatype: float
   type: sensor
   unit: km
-  description: Odometer reading, total distance travelled during the lifetime of the vehicle.
+  description: Odometer reading, total distance traveled during the lifetime of the vehicle.
 
-TripMeterReading:
+TraveledDistance:
   datatype: float
   type: sensor
   unit: km
-  description: Current trip meter reading.
+  description: Odometer reading, total distance traveled during the lifetime of the vehicle.
+
+TraveledDistanceSinceStart:
+  datatype: float
+  type: sensor
+  unit: km
+  description: Distance traveled since start of current trip.
+  comment: A new trip is considered to start when engine gets enabled (e.g. LowVoltageSystemState in ON or START mode).
+           A trip is considered to end when engine is no longer enabled.
+           The signal may however keep the value of the last trip until a new trip is started.
+
+StartTime:
```

Comment:
Changed from TimeSinceStart to StartTime based on discussion in last meeting.
Using string in UTC-format, as that is the format used in some other places in the tree to represent time - we currently do not use unix time stamps (i.e. seconds since 1970 represented as an int value anywhere)

(That is a side topic that comes up now and then, do we in VSS syntax need something that from a formal perspective tells that a signal contains a time stamp, either by using a predefined unit, or a datatype alias (for example "datatype:time" but specifying that it means string + iso 8601 syntax)